problems/isMultipleOfFive.js

```diff
@@ -13,7 +13,7 @@
 * returns: false (because the string "ten" is not a number)
 */
 
-function isMultipleOfFive() {
+function isMultipleOfFive(n) {
```

Comment:
mutliples are when a number can be divided by a number with no remainder. use the % to check for remainder.